decidim-bulletin_board-app/app/services/voting_scheme/dummy.rb

```diff
@@ -122,7 +122,7 @@ def process_tally_message(message_identifier, _message, content)
       results = build_questions_struct(0)
       state[:joint_shares].each do |question, answers|
         answers.each do |answer, joint_share|
-          results[question][answer] = (joint_share / state[:joint_election_key])**(1.0 / state[:trustees].count).round
+          results[question][answer] = ((joint_share / state[:joint_election_key])**(1.0 / state[:trustees].count)).round
```

Comment:
This looks like one of those 🐛 s that can take hours to identify 😄 
Nice catch! 🗜️